templates/services/merge/validator/lodestar.tmpl

```diff
@@ -1,39 +1,10 @@
 {{/* lodestar.tmpl */}}
 {{ define "validator" }}
-  validator-import:
-    container_name: validator-import-client
-    image: ${VL_IMAGE_VERSION}{{with .VlCustomCfg}}
-    depends_on:
-      config_consensus:
-        condition: service_completed_successfully{{end}}
-    networks:
-      - sedge
-    volumes:
-      - ${KEYSTORE_DIR}:/keystore
-      - ${VL_DATA_DIR}:/data{{if .VlCustomCfg}}{{if .VlRemoteCfg}}
-      - ./config.yml:/network_config/config.yaml{{end}}{{if .VlRemoteGen}}
-      - ./genesis.ssz:/network_config/genesis.ssz{{end}}{{if .VlRemoteDpl}}
-      - ./deploy_block.txt:/network_config/deploy_block.txt{{end}}{{end}}
-    command: |
-      validator import
-      --preset=${VL_LODESTAR_PRESET}{{if not .VlCustomCfg}}
-      --network={{if .SplittedNetwork}}${CL_NETWORK}{{else}}${NETWORK}{{end}}{{else}}
-      --paramsFile /network_config/config.yaml{{end}}
```

Comment:
I don' see this flag being set up in `ImportValidatorKeys` Lodestar's cmd

Reply:
Please check this commit https://github.com/NethermindEth/sedge/pull/208/commits/09743fd0d382943870ce2066823bbcdded15964a